addons/zylann.hterrain/hterrain_detail_layer.gd

```diff
@@ -30,11 +30,31 @@ const _API_SHADER_PARAMS = {
 	"u_ambient_wind": true
 }
 
-export(int) var layer_index = 0 setget set_layer_index, get_layer_index
-export(Texture) var texture setget set_texture, get_texture;
-export(float) var view_distance = 100.0 setget set_view_distance, get_view_distance
+# Minimum number of patches per grass instance.
+const MIN_PATCHES := 1
+# Maximum number of patches per grass instance.
+const MAX_PATCHES := 3
+
+export(int) var layer_index: int = 0 setget set_layer_index, get_layer_index
+export(Texture) var texture: Texture setget set_texture, get_texture;
+export(float) var view_distance: float = 100.0 setget set_view_distance, get_view_distance
 export(Shader) var custom_shader setget set_custom_shader, get_custom_shader
-export(float, 0, 10) var density = 4 setget set_density, get_density
+export(float, 0, 10) var density: float = 4 setget set_density, get_density
+export(int, 1, 3) var patches: int = 3 setget set_patches, get_patches
+
+func set_patches(new_value: int):
+	if not (MIN_PATCHES <= new_value && new_value <= MAX_PATCHES):
+		return
+	patches = new_value
+	# TODO Remove iteration of all chunks when Godot bug gets fixed
+	# See https://github.com/godotengine/godot/issues/32500
+	_multimesh = _generate_multimesh(CHUNK_SIZE, density, patches, null)
+	for k in _chunks:
+		var mmi = _chunks[k]
+		mmi.set_multimesh(_multimesh)
```

Comment:
Can we actually remove instances of this particular code since it seems to me that issue #32500 has been closed (fixed).